File: src_a_(C++)/Test4-4.c
//Проект Странник-Модула Для Windows 32, тестовая программа
//Группа тестов 4:ОПЕРАТОРЫ
//Тест номер    4:ЦИКЛ WHILE

include Win32
char s[15];

int i;
void main() {
  i=4;
  while(i<6)
    i=i+1;
  wvsprintf(s,"i=%li",&i);
  MessageBox(0,s,"i=6",0);
}

File: src_a_(Pascal)/Test4-4.pas
//Проект Странник-Модула Для Windows 32, тестовая программа
//Группа тестов 4:ОПЕРАТОРЫ
//Тест номер    4:ЦИКЛ WHILE
program Test4_4;
uses Win32;
var s:string[15];

var i:integer;
begin
  i:=4;
  while i<6 do
    i:=i+1;
  wvsprintf(s,"i=%li",addr(i));
  MessageBox(0,s,"i=6",0);
end.

